src/directive/sort.directive.ts

```diff
@@ -16,8 +16,9 @@
  See the License for the specific language governing permissions and
  limitations under the License.
  */
-import { Directive, Input, Output, Renderer, EventEmitter, ElementRef } from '@angular/core';
-import { JhiConfigService } from '../config.service';
+import {Directive, ElementRef, EventEmitter, Input, Output, Renderer} from '@angular/core';
```

Comment:
same here add the space between the bracket and the content

Reply:
it's done